### PR TITLE
add cv

### PR DESCRIPTION
**Pull request by maz-e**

### CODE REVIEW

File: staff/miguel-angel-zapatero/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV</title>
+</head>
+<body>
+    <header>
+        <section>
+            <!-- <hgroup> Por si quiero poner un subtitulo luego con cosas interesantes mias -->
```
Comment (lexbarba) [inline staff/miguel-angel-zapatero/cv/index.html]: @maz-e hgroup will be dropped soon, so we recommend you to not use it anymore. Here you have more info: https://www.w3.org/html/wg/wiki/ChangeProposals/hgroup

File: staff/miguel-angel-zapatero/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                <p>Inglés (First Certificate Diciembre del 2014)</p>
+            </section>
+            <section>
+                <h1>INTERESES</h1>
+                <p>Escalada y montaña</p>
+                <p>Tecnología y videojuegos</p>
+            </section>
+        </aside>
+    </article>
+    <footer></footer>
```
Comment (lexbarba) [inline staff/miguel-angel-zapatero/cv/index.html]: @maz-e it makes no sense to open and close a tag with no data inside. Please fill or remove. Thanks!

**Review by lexbarba — CHANGES_REQUESTED** [review]

File: staff/miguel-angel-zapatero/cv/index.html (hunk 1 above)
Comment (maz-e) [reply staff/miguel-angel-zapatero/cv/index.html]: @lexbarba Hi Alex! all hgroup removed, thanks!

File: staff/miguel-angel-zapatero/cv/index.html (hunk 2 above)
Comment (maz-e) [reply staff/miguel-angel-zapatero/cv/index.html]: @lexbarba footer removed! 👍

**Pull request merged**